transforms/.make.transforms

```diff
@@ -293,7 +293,7 @@ conventions: .transforms.check_required_macros
 .transforms.minio-load::
 	@# Help: Install the test-data/input files into minio for $(TRANSFORM_NAME)
 	dir=$$(find $(REPOROOT)/transforms -type d -name $(TRANSFORM_NAME));	\
-	$(MAKE) MINIO_SRC=$$dir/test-data/input MINIO_DEST=$(TRANSFORM_NAME)/ .defaults.minio.load-test-data
+	$(MAKE) MINIO_SRC=$$dir/ray/test-data/input MINIO_DEST=$(TRANSFORM_NAME)/ .defaults.minio.load-test-data
```

Comment:
`ray` should probably be changed to `$(TRANSFORM_RUNTIME)` to be runtime-independent.  The value of this macro is the name of the directory. one of python, ray or spark.

Reply:
Just made it work. Moving forward, yes

Reply:
unclear why you won't make this change like you did for the comments?  The time is now :)